src/test/java/hw1/DivisionTest.java

```diff
@@ -0,0 +1,75 @@
+package hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class DivisionTest {
+
+    private Calculator calc = new Calculator();
+
+    //    @BeforeTest
+    //    public void before_class(){
+    //         calc = new Calculator();
+    //    }
```

Comment:
This is the right approach, I mean `@BeforeSomething`

Could you please update unit under test initialization to this form

Reply:
done